docs/docs-components/COMPONENT_DATA.js

```diff
@@ -3360,6 +3309,57 @@ const BUILDING_BLOCKS_LIST: $ReadOnlyArray<ListItemType> = [
       figma: null,
     },
   },
+  {
+    svg: <Popover />,
```

Comment:
Noting the purpose of this move in the PR would've been helpful — this file is not easy to read, especially in the GH web interface